apps/mealplanr/src/components/recipes/Recipes.tsx

```diff
@@ -76,18 +88,29 @@ const recipeInfo = (id: number) => {
 };
 
 // Handle to set placholder image when src is unavailable
-function handleAltImg(e: any) {
-	e.target.src='/alt.png';
-	e.target.parentNode.classList.remove("shadow"); // Shadow is applied to the path of svg - we do not want that
+function handleAltImg(e: React.SyntheticEvent<HTMLImageElement, Event>) {
+	const target = e.target as HTMLImageElement;
+	const ptarget = target.parentElement as HTMLDivElement;
+	target.src='/alt.png';
```

Comment:
This and anything else related to the Public folder needs to be added in a new story (issue).

Reply:
New issue created: #120